Fix up circleci tests

chrisgilmerproj opened the pull request:
This should get circleci to run.

SteveKrzysiak commented:
I modeled this after the other kms modules, does this need to be tweaked for them as well?

chrisgilmerproj commented:
Yes and no. You forgot to set this up with CircleCI and when I did so (because I was setting up other modules) I noticed that the tests were failing. You can see the issues fixed in the `README.md`. While I was here I updated the version of terraform-docs to the latest, which is always a good idea, and added pre-commit configuration which helps me manage these correctly.
